install_scripts/install_mbot_dependencies.sh

```diff
@@ -22,6 +22,9 @@ apt -y install python3-dev python3-numpy python3-matplotlib python3-opencv pytho
 # Install python pkgs for MBot OLED
 apt -y install python3-qrcode python3-luma.oled
 
+# Install firmware upload script
+sudo cp mbot-upload-firmware /usr/local/bin
```

Comment:
Right now, if you run this script, it installs `mbot-upload-firmware` but it would not work unless you run the other picotools install script, which would be confusing for the reader.

Two options:
1. Move installing the script to the end of `install_scripts/install_picotool.sh` script. You could rename it to `install_mcb_tools.sh` for "MBot Control Board" or something similar.
2. Move all the picotools setup from `install_scripts/install_picotool.sh` into this script.
I prefer 2 because it doesn't add an extra step that could be skipped, and I can't think of a case where the user would want everything in this script but not this upload script. I'll leave it up to you though!

Reply:
yup I agree option2 is better to reduce human error, will push soon